emoji/emoji_transfer_service.py

```diff
@@ -1,4 +1,4 @@
-from emoji import image_client
+from image import image_client
```

Comment:
I'm thinking it may be time to make a class here, but no need to hold it up right now